getSiteInfo.py

```diff
@@ -199,50 +201,77 @@ def getLFNAndPFNFromPhEDEx(site, debug = False):
 def addInformationNotInSiteDB(site, debug = False):
     if site.alias in siteDBDict:
         site.local_redirector    = siteDBDict[site.alias][0] if siteDBDict[site.alias][0]!='' else "None"
-        site.xrootd_endpoint     = siteDBDict[site.alias][1] if siteDBDict[site.alias][1]!='' else "None"
-        site.gsiftp_endpoint     = siteDBDict[site.alias][2] if siteDBDict[site.alias][2]!='' else "None"
-        site.local_path_to_store = siteDBDict[site.alias][3] if siteDBDict[site.alias][3]!='' else "None"
+        site.local_path_to_store = siteDBDict[site.alias][2] if siteDBDict[site.alias][2]!='' else "None"
+        site.gsiftp_endpoint     = site.pfn
+        if siteDBDict[site.alias][1]!='':
+            site.xrootd_endpoint = siteDBDict[site.alias][1]
+        else:
+            site.create_xrootd_endpoint()
+    else:
```

Comment:
i think this logic could be simplified:
```python
if site.alias in siteDBDict:
    ...
    site.xrootd_endpoint = siteDBDict[site.alias][1]
if site.xrootd_endpoint == '':
    site.create_xrootd_endpoint()
```
i.e. since you're going to create an endpoint whenever the value is empty, you don't have to check emptiness several times.

Reply:
I think you'll agree that the new version has the same functionality, but fewer lines of code.